python/lsst/ts/atmonochromator/monochromator_csc.py

```diff
@@ -538,12 +544,24 @@ async def health_monitor_loop(self) -> None:
                 await self.tel_loopTime.set_write(loopTime=curr_tai - start_tai)
                 await asyncio.sleep(self.heartbeat_interval)
             except Exception:
```

Comment:
Is this changing anything? doesn't seem like there's an behavior change here. Do you understand why this is not working the way it is?